packages/camera/camera_android_camerax/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.7.0
```

Comment:
I did a minor version bump so that I can constraint the new package/camera version to depend on this on newer, since the rotation logic will change between 0.6.5+5 and 0.7.0